src/main/java/com/googleinterns/zoomtube/transcriptParser/TranscriptParser.java

```diff
@@ -107,7 +107,8 @@ private URL getTranscriptUrlForVideo(String videoId) throws IOException {
    * @param url Indicates the url to fetch and parse.
    * @throws IOException if there is an error parsing the transcript.
    */
-  private Document fetchUrlAsXmlDocument(URL url) throws IOException {
+  // TODO: Move this to a Utils method.
+  public static Document fetchUrlAsXmlDocument(URL url) throws IOException {
```

Comment:
instead of marking this as a TODO, you should have create the PR to do this before sending out this PR.

Reply:
gotcha, I'll remove this TODO then because I don't think I'll be able to get to it with the time constraints.

Reply:
No, please keep the TODO because this indicates this needs to get done.

Reply:
gotcha, re-added